qa/suites/rbd/cache/workloads/fio.yaml

```diff
@@ -0,0 +1,32 @@
+tasks:
+- install:
+    extra_packages: ['ceph-immutable-object-cache']
+- ceph:
+    conf:
+      osd:
+        osd shutdown pgref assert: true
+      client:
+        rbd_parent_cache_enabled: true
+        #debug immutable_obj_cache: 20/20
+        immutable object cache path: /tmp/ceph-immutable-object-cache
+        immutable object cache max size: 50G
+      global:
+        auth cluster required: none
+        auth service required: none
+        auth client required : none
+- exec:
+    client.0:
+      - ceph-immutable-object-cache -b
```

Comment:
Nit: don't start the daemon directly. Have the associated task start and manage the daemon.